package.json

```diff
@@ -37,47 +39,63 @@
     "node": ">=6.0"
   },
   "scripts": {
-    "coverage": "nyc npm run test && nyc report --reporter=text-lcov > .nyc_output/lcov.info",
-    "coveralls": "npm run coverage && coveralls <.nyc_output/lcov.info",
-    "build": "babel src -d lib",
-    "integration": "tape -r babel-register test/integration/*.js",
-    "lint": "standard",
-    "prepublish": "npm run build",
-    "test": "npm run lint && npm run unit && npm run integration",
-    "unit": "tape -r babel-register test/*.js"
+    "build": "ethereumjs-config-build",
+    "prepublishOnly": "npm run test && npm run build",
+    "coverage": "ethereumjs-config-coverage",
+    "coveralls": "ethereumjs-config-coveralls",
+    "docs:build": "typedoc --out docs --mode file --readme none --theme markdown --mdEngine github --excludeNotExported src",
+    "format": "ethereumjs-config-format",
+    "format:fix": "ethereumjs-config-format-fix",
+    "tslint": "ethereumjs-config-tslint",
+    "tslint:fix": "ethereumjs-config-tslint-fix",
+    "tsc": "ethereumjs-config-tsc",
+    "lint": "ethereumjs-config-lint",
+    "lint:fix": "ethereumjs-config-lint-fix",
+    "test": "ts-node node_modules/tape/bin/tape ./test/index.ts"
   },
   "dependencies": {
     "babel-runtime": "^6.11.6",
     "bl": "^1.1.2",
     "debug": "^2.2.0",
+    "ethereumjs-tx": "^1.3.7",
     "inherits": "^2.0.1",
     "ip": "^1.1.3",
     "k-bucket": "^3.2.1",
     "keccak": "^1.0.0",
-    "lru-cache": "^4.0.1",
+    "lru-cache": "^5.1.1",
     "ms": "^0.7.1",
     "rlp-encoding": "^3.0.0",
     "safe-buffer": "^5.1.1",
     "secp256k1": "^3.1.0"
   },
   "devDependencies": {
+    "@ethereumjs/config-nyc": "^1.1.1",
+    "@ethereumjs/config-prettier": "^1.1.1",
+    "@ethereumjs/config-tsc": "^1.1.1",
+    "@ethereumjs/config-tslint": "^1.1.1",
+    "@types/async": "^2.4.1",
+    "@types/bl": "^2.1.0",
+    "@types/chalk": "^2.2.0",
+    "@types/debug": "^4.1.4",
+    "@types/ethereumjs-tx": "^1.0.1",
+    "@types/ip": "^1.1.0",
+    "@types/lru-cache": "^5.1.0",
+    "@types/ms": "^0.7.30",
+    "@types/secp256k1": "^3.5.0",
```

Comment:
the type definitions have changed in the meantime and are not matching: would be best to pin to `3.5.0`

Reply:
```suggestion
    "@types/secp256k1": "3.5.0",
```